tests/libcxx/expected_results.txt

```diff
@@ -561,6 +561,9 @@ std/language.support/support.limits/support.limits.general/algorithm.version.com
 std/language.support/support.limits/support.limits.general/functional.version.compile.pass.cpp FAIL
 std/language.support/support.limits/support.limits.general/iterator.version.compile.pass.cpp FAIL
 
+# libc++ doesn't implement P2588R3 barrier's Phase Completion Guarantees

```

Comment:
Can we please move this up into the first "issues upstream knows about" section? Upstream knows they need to implement proposals that have merged into the working draft. (Ditto for `skipped_tests.txt`, of course.)